### DISCUSSION TITLE
Shelly Wave 1 mini

### DISCUSSION THREAD

**Discussion by skywalker1979** [zwave-js/zwave-js-ui#3988] [Q&A] [answered]
hi...just a quick question... i hope to not cause too much fuss.
Shelly Wave 1 mini....in the website ( https://kb.shelly.cloud/knowledge-base/wave-1-mini ) it says:

```
1x click: Change the state of the output state O (O1) to the opposite one and send command to the associated devices in associated groups 2 and 3 (check chapter [Z-Wave Association](https://kb.shelly.cloud/knowledge-base/wave-1-mini#Z-Wave%E2%84%A2-Associations))

2x click: If the delay between first in second click is less then 500ms, this is interpreted as double clik. Send command to the associated devices (dimmers, shutters,….) in associated groups 2 and 3 (check chapter [Z-Wave Association](https://kb.shelly.cloud/knowledge-base/wave-1-mini#Z-Wave%E2%84%A2-Associations))

Hold: Send command to the associated devices in associated group 3 (check chapter [Z-Wave Association](https://kb.shelly.cloud/knowledge-base/wave-1-mini#Z-Wave%E2%84%A2-Associations))

Release: Send command to the associated devices in associated group 3 (check chapter [Z-Wave Association](https://kb.shelly.cloud/knowledge-base/wave-1-mini#Z-Wave%E2%84%A2-Associations))

```

Is this the "Scene" function like fibaro has? i'm not able to make it work so i don't this it is.
If not, can the 2xclick and hold be used on Z Wave js ui when using Wave 1 mini?

Thanks and have a nice day!

**Comment by robertsLando** [comment]
No clue, @AlCalzone ?

**Comment by AlCalzone** [comment, marked as answer]
The website explains the mentioned associations further down:
![grafik](https://github.com/user-attachments/assets/6f9697da-e8dc-4df7-a620-731986bbc0cb)

So, those aren't really scenes, just commands sent to other devices.

If you add the controller to groups 2 and 3, it should receive those.
The commands sent in group 2 may appear under the values for Basic CC - not entirely sure if this needs an additional compat flag
The commands sent in group 3 are exposed as events - see node events tab.

**Comment by skywalker1979** [comment]
Got it...thanks...i was hoping it worked with the scene function in domoticz....
Thanks for the quick reply...
